src/olympia/constants/scanners.py

```diff
@@ -7,3 +10,12 @@
     WAT: 'wat',
     YARA: 'yara',
 }
+
+# Action IDs are also used for severity (the higher, the more severe).
+NO_ACTION = 1
+FLAG_FOR_HUMAN_REVIEW = 20
+
+ACTIONS = {
+    NO_ACTION: _('no action'),
+    FLAG_FOR_HUMAN_REVIEW: _('flag for human review'),
```

Comment:
I was thinking about adding the rest when we'll implement them, but maybe we can add all options now.

Reply:
I think that's fine as-is, we can add to the dict as we implement the actions. I started filing issues but we're going to be blocked soon partially (or at least applying actions to *unlisted* submissions will be blocked)

Reply:
Thanks. I see that the unlisted submissions won't be as straightforward as we initially thought indeed :(